core/arch/arm/plat-rcar/romapi.c

```diff
@@ -129,7 +130,13 @@ uint32_t plat_rom_getrndvector(uint8_t rndbuff[PLAT_RND_VECTOR_SZ],
 	assert(rndbuff_pa % RCAR_CACHE_LINE_SZ == 0);
 	assert(scratch_pa % RCAR_CACHE_LINE_SZ == 0);
 
-	ret = plat_call_romapi(func_addr, rndbuff_pa, scratch_pa, scratch_sz);
+	while (try++ < 3) {
+		ret = plat_call_romapi(func_addr, rndbuff_pa, scratch_pa,
+				       scratch_sz);
+		if (ret == 0)
+			break;
+		IMSG("ROM_GetRndVector() returned 0x%X", ret);
```

Comment:
`"%#"PRIx32`, gives `0x` prefix automatically and `PRIx32` for a 32-bit type using lower case letters.